src/app/components/event-page/event-page.component.scss

```diff
@@ -156,9 +156,20 @@
     margin-right: 5px;
 }
 
+.join-button-active {
+    background-color: $join-green;
+    margin-left: 25px;
+    transform: scale(1.35);
+}
+
+.join-button-disabled {
+    background-color: $join-disabled;
+    margin-left: 25px;
+    transform: scale(1.35);
+}
```

Comment:
you should have a  common class action-button that has:
margin-left: 25px;
transform: scale(1.35);
now you have this code in 3 different classes

And in the other 3 classes set only the color for each button. (join-button-active, join-button-disabled, leave-button)